internal/client/clustergroup/iam_policy/iam_resource.go

```diff
@@ -0,0 +1,73 @@
+/*
+Copyright © 2022 VMware, Inc. All Rights Reserved.
+SPDX-License-Identifier: MPL-2.0
+*/
+
+package iamclustergroupclient
+
+import (
+	"fmt"
+
+	"github.com/vmware-tanzu/terraform-provider-tanzu-mission-control/internal/client/transport"
+	clustergroupmodel "github.com/vmware-tanzu/terraform-provider-tanzu-mission-control/internal/models/clustergroup"
+	iammodel "github.com/vmware-tanzu/terraform-provider-tanzu-mission-control/internal/models/iam_policy"
+	clustergroupiammodel "github.com/vmware-tanzu/terraform-provider-tanzu-mission-control/internal/models/iam_policy/clustergroup"
+)
+
+// New creates a new cluster group IAM policy resource service API client.
+func New(transport *transport.Client) ClientService {
+	return &Client{Client: transport}
+}
+
+/*
+  Client for cluster group IAM policy resource service API.
+*/
+
+type Client struct {
+	*transport.Client
+}
+
+// ClientService is the interface for Client methods.
```

Comment:
ClientService is the interface for ManageV1alpha1ClusterGroupIAMPolicy Client methods.